src/main/resources/common/sitemap-entry.jsp

```diff
@@ -24,7 +24,7 @@
     <c:set var="serverName" value="${sitemap:getServerName(urlHostServerName)}"/>
 
     <jcr:nodeProperty var="lastModified" node="${urlNode}" name="jcr:lastModified"/>
-
+    <!--${urlNode}-->
```

Comment:
As the goal is to debug the execution, can't we use a parameter to enable that comment ? as well, you miss all nodes that are `jseomix:noIndex`
Moreover can you add an explanation about the value itself?

Reply:
Thanks David, done